website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -46,34 +56,8 @@ By default this will be the namespace that the operator is running in. To watch
 mgr, err := ctrl.NewManager(cfg, manager.Options{Namespace: ""})
 ```
 
-It is also possible to use the [MultiNamespacedCacheBuilder][multi-namespaced-cache-builder] to watch a specific set of namespaces:
-```Go
-var namespaces []string // List of Namespaces
-// Create a new Cmd to provide shared dependencies and start components
-mgr, err := ctrl.NewManager(cfg, manager.Options{
-   NewCache: cache.MultiNamespacedCacheBuilder(namespaces),
-})
-```
-
-#### Operator scope
-
 Read the [operator scope][operator_scope] documentation on how to run your operator as namespace-scoped vs cluster-scoped.
 
-### Multi-Group APIs
```

Comment:
moved to reference since we do not use multigroup in the example and 80% of the uses probably never will use need this support/feature.